src/bin/spin.rs

```diff
@@ -50,7 +50,7 @@ enum SpinApp {
     New(NewCommand),
     Add(AddCommand),
     Up(UpCommand),
-    #[clap(subcommand)]
+    #[clap(subcommand, hide = true)]
```

Comment:
This seems orthogonal to adding registry support, and I thought we were going to allow a bit of time for deprecation.  For 0.9 we should maybe print a deprecation message; I wouldn't necessarily entwine that with this PR though.

Reply:
Yeah, splitting the PRs seems like a good idea to me.